otp.py

```diff
@@ -0,0 +1,130 @@
+"""Python Project to Generate and Validate OTPs"""
+import math
+import random
+from datetime import datetime
+import vonage
+import constant
+from otp_database import DatabaseOtp
+
+
+class MobileNumberMisMatch(Exception):
+    """Mobile Number Mismatch Custom Error if Number length is not 10 and number is not numeric"""
+
+    def __init__(self, message: str) -> None:
+        """
+        custom error class to throw mobile Number Mismatch Error
+        @param message: str
+        """
+        self.message = message
```

Comment:
from the next time whenever you are write custome exception do not wirte in the same file move to utils. or ask @yatish.prajapati sir either it's right or wrong..